styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```

Comment:
- [ ] As I mentioned in my previous `inline-comment` please add the class for your header and then use the class for styling because it will help you in future to work with the given instructions and make you the kind of developer who really pays attention on the small details.

## Requirement

![image](https://github.com/nouman17610/hello-microverse/assets/66476329/6a28658a-dd14-49dd-981b-fb3754bc596d)